src/partials/messenger.conversation.html

```diff
@@ -13,11 +13,11 @@
         <div class="header-details" ng-click="ctrl.showReceiver()">
             <div class="conversation-header-details-name"
                  ng-bind-html="ctrl.receiver.displayName | escapeHtml | emojify"></div>
-            <div class="conversation-header-details-detail" ng-if="ctrl.type == 'contact'">
-            <eee-verification-level ng-if="ctrl.type == 'contact'"
+            <div class="conversation-header-details-detail" ng-if="ctrl.type === 'contact'">
+            <eee-verification-level ng-if="ctrl.type === 'contact'"
                                     contact="ctrl.receiver"></eee-verification-level>
             </div>
-            <div class="conversation-header-details-detail" ng-if="ctrl.type == 'group'"
+            <div class="conversation-header-details-detail" ng-if="ctrl.type !== 'contact'"
```

Comment:
I'd be conservative here and write `ctrl.type === 'group' || ctrl.type === 'distributionList'` :slightly_smiling_face: